rm/rm-server/build.gradle

```diff
@@ -21,8 +21,8 @@ dependencies {
     testCompile "org.objectweb.proactive:programming-extension-pnpssl:${programmingVersion}"
     testCompile project(':common:common-api').sourceSets.test.output
 
-    testCompile 'org.jruby:jruby-complete:9.0.5.0'
-    testCompile 'org.python:jython-standalone:2.7.0'
+    testCompile 'org.jruby:jruby-complete:9.1.12.0'
+    testCompile 'org.python:jython-standalone:2.7.1'
```

Comment:
Yes you are right, I looked through the jython news and could find contradictory information:

Jython news: https://hg.python.org/jython/rev/0df7adb1b397 (2.7.1 release is a proper release)
Maven Repository: https://mvnrepository.com/artifact/org.python/jython (2.7.1b3 is the latest)
I think it is safe to use 2.7.1, but mvnrepository.com doesn't seem to be up to date.